### DISCUSSION TITLE
It's possible to revert Xbox One controllers to older firmware for compatibility with MissionControl.

### DISCUSSION THREAD

**Discussion by Renniechris1** [ndeadly/MissionControl#660] [General]
I'm linking an guide from Microsoft on how to revert your Xbox One Bluetooth controllers to an older firmware for wider compatibility with none Xbox devices. This will enable those controllers to work with MissionControl once again. The guide can be followed on both an Xbox or Windows device.

https://support.xbox.com/en-GB/help/hardware-network/accessories/controller-firmware-reversion#:~:text=Use%20the%20Xbox%20Accessories%20app,issues%20after%20the%20last%20update.
![image](https://github.com/ndeadly/MissionControl/assets/107317008/abd22a63-890f-46d9-81b3-6dbe1e07d213)

**Comment by Masamune3210** [comment]
Mind you, this will not work if your controller came out of the box with LE firmware. It will only downgrade the controller's firmware if the controller has been updated to the LE firmware. Basically, if the box or the controller said for Series consoles on it anywhere, you are likely out of luck, but if its a legacy Xbox One controller, this will likely get you back on track.

**Reply by Renniechris1** [reply]
Exactly right. Only Xbox One controllers with Bluetooth were ever compatible and are able to be rolled back in this way.

**Reply by rarefishinvestor** [reply]
I just bought a controller specifically for use with the switch/mission control. It came with the LE firmware out of the box. Is it possible to install the old firmware or should I just send it back?

**Reply by Renniechris1** [reply]
As far as I'm aware as long as it's an Xbox One controller and not a Xbox Series S/X controller you should be able to downgrade the firmware  with the above methods and use it with mission control (easiest way to tell what model you have besides packaging is the share button on Series controllers thats not present on earlier Xbox One models)

**Reply by ndeadly** [reply]
> I just bought a controller specifically for use with the switch/mission control. It came with the LE firmware out of the box. Is it possible to install the old firmware or should I just send it back?

Any controller sold with an LE firmware will be the newer Series X|S variety. You can't downgrade those

**Reply by euaaron** [reply]
Even my Xbox One Controller 1708 is having issues to downgrade/revert firmware to stock version...

This is what happens:

![image](https://github.com/ndeadly/MissionControl/assets/26356962/2a0cac17-6a8c-422f-996f-29c3f2e9f7e0)

![image](https://github.com/ndeadly/MissionControl/assets/26356962/603b9800-6089-414f-a63c-bf98cbe3ef41)

![image](https://github.com/ndeadly/MissionControl/assets/26356962/00f5bd86-c287-41c3-8c5b-10d3d21dfd65)

Maybe the best way to get it working with mission control would be to buy a new Xbox One controller, or a PS4/5 Dualsense.

**Reply by ndeadly** [reply]
Try with a different cable or PC. I've heard of at least one case where this allowed the downgrade

I can recommend Sony controllers as probably the best overall if you're looking to buy something. Just be sure to stay away from counterfeits, as they often don't work properly.

**Reply by Selaphane** [reply]
I am having the exact same issue with the exact same controller... I've tried:

1. Uninstalling the device driver, reconnecting controller and re-installing multiple times
2. Disabling Xbox extended feature support in Steam, trying to revert, failing, then turning it back on in Steam and still failing
3. Running the xboxaccessories:\\\firmwareupdate?legacyDowngrade=true command in the run box, as well as using the "Check your controller's firmware" link on this page https://support.xbox.com/en-US/help/hardware-network/accessories/controller-firmware-reversion
4. Uninstalling and re-installing the Xbox accessories app, also have reset it in app settings multiple times
5. Two different PCs, as well as like 10 different micro USB cables

I'm honestly completely stumped on what to do outside of just buying a new controller and never updating it I guess...

**Reply by ndeadly** [reply]
Are you sure it's the exact same controller? What model number is printed inside the battery compartment? Which firmware version is reported by the Xbox accessories app?

**Reply by euaaron** [reply]
I believe she was saying it is the exact same controller as mine, which is model 1708. And so as me, she is unable to downgrade the controller's firmware.

I also tried all the steps she mentioned but there's one thing I haven't tried, which is attaching the controller to an Xbox and trying to downgrade from it instead of using a PC.

**Reply by Selaphane** [reply]
Yes I have model 1708. Xbox Accessories app reports firmware 5.20. I actually think the recent update to the Xbox Accessories app is causing this problem. It was updated 5 days ago, and ever since then this has become a problem.

Sadly, I do not own an Xbox so I am unable to try reverting that way.

**Reply by Tartifless** [reply]
Same issue, was able to revert a few months ago, did the error of updating again, now impossible to revert anymore !

**Reply by hactesp** [reply]
The same with me. I update xbox one s and xbox elite 2 and now could not revert anymore ?

**Reply by sakej** [reply]
I can confirm that the controllers model 1914 (I have two of them bought at different times) are NOT compatible with the official firmware downgrade...
Anyone know is there is any unofficial way to do so?

**Reply by ndeadly** [reply]
Nope. It may not even be possible, since these controllers never had a pre-LE firmware. Outside of people downgrading for use with Mission Control, I don't expect the demand for such a thing to be great enough for anyone to spend their time on it.

**Comment by DARKFiB3R** [comment]
How to do this via PC...

On your Windows device, make sure the Xbox Accessories app is installed. (Available at https://aka.ms/xboxaccessoriesupdateapp )

Connect your controller via USB

Press the Windows key + R to open the open the Run dialog box.

Copy and paste the link below and hit Enter.

xboxaccessories://firmwareupdate?legacyDowngrade=true

This will open the controller firmware reversion screen in the Xbox Accessories app on your device.

Click the Revert button.

https://www.thewindowsclub.com/how-to-rollback-xbox-controller-firmware-update

Thanks for pointing me in the right direction :)

**Reply by HerobrineTV** [reply]
Thanks, worked on My Elite Controller Series 2

**Reply by euaaron** [reply]
I would love it had worked with my 1708 controller... 😢

**Reply by DARKFiB3R** [reply]
You could try contacting Obinna about his article, and see if he can help.

https://www.thewindowsclub.com/how-to-rollback-xbox-controller-firmware-update

https://ng.linkedin.com/in/chidumosobalu

**Reply by zYn-Hazard** [reply]
yep this helped me too, i bought a Pre-Owned xbox controller 1708 model firmware version was 5.2.0 etc.. and was not connecting on my PC via Bluetooth adapter, after following the steps above fixed it.

**Reply by euaaron** [reply]
> I would love it had worked with my 1708 controller... 😢

I tried again last month and this time it worked!
Looks like it was failing to downgrade due to something wrong with the cable, cuz after buying a new one and trying again it worked.

**Comment by iliaakm** [comment]
Yes, it's completely possible, use [this guide](https://drive.google.com/drive/folders/1OYuGXa89gJIlrbUQIrF4JB-4tNjA7egV) by @Vizr_oo from reddit

**Reply by DARKFiB3R** [reply]
Why has Vizr_oo been banned from Reddit?

**Reply by iliaakm** [reply]
@DARKFiB3R no idea

**Reply by gozerthedestroyer** [reply]
This is the correct and best answer when it come to Xbox Model 1914 Controllers. Recently I used the updated Xbox Accessories App on Windows 11 and updated my Xbox Model 1914 Controller thinking the firmware update would only make it better and more stable and i was very wrong! After the update my controller no longer would wirelessly connect to anything it previously did with bluetooth and i was only able to get functionality of the controller through a USB connection. Then, Everywhere online i looked about this issue said it was a firmware issue but it impossible to revert the controllers firmware without an Xbox and even after trying my friends Xbox the firmware still would not revert. Finally after coming along to this post i gained the solution! Upon going to [this link(same as above)](https://drive.google.com/drive/folders/1OYuGXa89gJIlrbUQIrF4JB-4tNjA7egV) and following the instructions in the ReadMe.txt (At the bottom is the instruction to revert your firmware), I was able to revert my controllers firmware and was able to restore Bluetooth connectivity back to my Xbox Model 1914 Controller!
